inference/core/workflows/core_steps/visualizations/polygon_zone/v1.py

```diff
@@ -84,7 +85,7 @@ def get_execution_engine_compatibility(cls) -> Optional[str]:
 class PolygonZoneVisualizationBlockV1(VisualizationBlock):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._cache: Dict[str, np.ndarray] = {}
+        self._cache: OrderedDict[str, np.ndarray] = OrderedDict()
```

Comment:
we have LRUCache which we usually use in workflow blocks, see e.g.
 - https://github.com/roboflow/inference/blob/main/inference/core/workflows/core_steps/models/foundation/clip/v1.py#L97
  - https://github.com/roboflow/inference/blob/main/inference/core/workflows/core_steps/visualizations/grid/v1.py#L97

Reply:
but essentially same operations.  Might be good to standarize (e.g. might need to add locking to LRUCache now that I think about it with recent async changes)